tests/stories/Example.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
+  import type { Snippet } from 'svelte';
   import type { HTMLButtonAttributes } from 'svelte/elements';
 
-  interface Props extends HTMLButtonAttributes {
+  type Props = {
```

Comment:
Was the reason for this change _(from `interface` to `type`)_ to reduce the amount of props being inferred?

Reply:
you're right, after I fixed all the types, I didn't actually need to narrow it down anymore. reverted.